app/core/search.py

```diff
@@ -1227,7 +1227,7 @@ def process_vespa_search_response(
 ) -> SearchResponse:
     """Process a Vespa search response into a F/E search response"""
     return SearchResponse(
-        hits=vespa_search_response.total_hits,
+        hits=len(vespa_search_response.families),
```

Comment:
Are we still happy referring to this as 'hits'?

Reply:
this actually matches the current Opensearch definition of hits, and @harrisonpim is raising a ticket in data science to look into making "hits" a more accurate measure of the number of families that would match rather than cutting it of at the query limit